### PR TITLE
[fix] (udf) fix check_fn and fn_call function name not same

### PR DESCRIPTION
**Pull request by yangzhg**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

fix check_fn and fn_call function name not same

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (No Need)
3. Has document been added or modified: (No Need)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: gensrc/proto/function_service.proto [hunk 1]
```diff
@@ -35,8 +35,8 @@ message PFunctionCallRequest {
 }
 
 message PFunctionCallResponse {
-    optional PValues result = 1;
-    optional PStatus status = 2;
+    required PValues result = 1;
```
Comment (morningman) [inline gensrc/proto/function_service.proto:38]: why changing to requried?

Comment (stalary) [reply gensrc/proto/function_service.proto:38]: During the test, I found that forgetting to return would lead to be crash

Comment (yangzhg) [reply gensrc/proto/function_service.proto]: fixed

**Comment by yiguolei** [comment]
It's better to forbidden use required field in Thrift or ProtocolBuffer. We could not remove it in the future and there will be a lot of rubbish code. For example, version hash, schema hash.... It is too terrible!!!!!

**Comment by yangzhg** [comment]
> It's better to forbidden use required field in Thrift or ProtocolBuffer. We could not remove it in the future and there will be a lot of rubbish code. For example, version hash, schema hash.... It is too terrible!!!!!

I have removed required

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**